apps/web/modules/event-types/views/event-types-listing-view.tsx

```diff
@@ -513,19 +513,21 @@ export const EventTypeList = ({
                                   </DropdownItem>
                                 </DropdownMenuItem>
                               )}
-                              {!isManagedEventType && !isChildrenManagedEventType && (
-                                <>
-                                  <DropdownMenuItem className="outline-none">
-                                    <DropdownItem
-                                      type="button"
-                                      data-testid={`event-type-duplicate-${type.id}`}
-                                      StartIcon="copy"
-                                      onClick={() => openDuplicateModal(type, group)}>
-                                      {t("duplicate")}
-                                    </DropdownItem>
-                                  </DropdownMenuItem>
-                                </>
-                              )}
+                              {(group.metadata?.readOnly === false || group.metadata.readOnly === null) &&
```

Comment:
I think it would make more sense to avoid using metadata to solve this issue and actually check for the role of the member within the organization

Reply:
`readOnly` at backend is computed based on user membership only. But metadata isn't needed here. There is a `readOnly` variable already which is used  to give similar treatment to other actions.

Reply:
See line https://github.com/calcom/cal.com/pull/15515/files#diff-46c5891907ea23858fe4929181c3a92703d3410afa5b98b8ad8b8e4d01b83aa6L505